README.md

```diff
@@ -46,117 +36,75 @@ From our [documentation](https://docs.sentry.io/platforms/react-native/), the fo
 7. `cd ios && pod install`
 8. Optional - Create a .env file in project root and add `SE=<value>`
 
-Don't forget to bump your release version depending on platform
```

Comment:
In the list above, the number 5 needs an update.
> 5. If you don't have cocoapods or get an error about cocopods out of date, run `gem install cocoapods`

```
5. If you don't have cocoapods or get an error about cocopods out of date, run `bundle install` and `pod repo update`
```

Reply:
added it, thanks.